guides/data/models.md

```diff
@@ -22,18 +22,18 @@ If your app can swap databases with zero refactoring, that either means:
 
 1. The underlying databases are identical in their querying and storage
 semantics (rarely the case), or
-2. You were using some lowest common denominator of querying and storage semantics for the two databases that is
+2. You were using some lowest common denominator of querying and storage semantics for the two databases that are
```

Comment:
Actually I think the original is correct - the subject of that verb is "lowest common denominator" (singular). The "two databases" is a qualifier of that subject, rather than the subject itself.

Perhaps that whole sentence is confusing though ...

Reply:
@davewasmer The singular verb is does not appear to agree with the plural subject `the two databases`. I would consider changing the verb form for subject-verb agreement.

Reply:
I might have just not had enough coffee this morning, but after re-reading, I'm pretty sure the subject of that verb is `lowest common denominator`, which is singular. I think it's clearer if I remove the qualifying clause:

```
You were using some lowest common denominator <snip> that is equivalent ...
```

Does that make sense?